### PR TITLE
feat: support Blender3.6.16, Bledner3.6.17

### PR DESCRIPTION
**Pull request by AoiKamishiro**
Add support of Blender3.6.16LTS & Blender3.6.17LTS.

### CODE REVIEW

File: src/dll/BlenderPatchPatterns.h [hunk 1]
```diff
@@ -346,6 +346,24 @@ static std::unordered_map<std::string, BlenderPatchPattern> Patchers{
...
+        "3.6.16",
+        {
+            "E8 EC B5 41 00",
+            "4C 8D 05 19 08 00 00",
+            "E8 7B FF FF FF 84 C0",
+            "E8 EB BD 00 00 48 85 C0",
+        },
+    },
+    {
+        "3.6.16",
```
Comment (mika-f) [inline src/dll/BlenderPatchPatterns.h]: This line probably a typo?

```suggestion
        "3.6.17",
```

**Review by mika-f — COMMENTED** [review]
I commented on the PR, maybe it's an unintended change, please check it out.

**Comment by AoiKamishiro** [comment]
I forgot to rewrite the version number when I copied and pasted because it was the same value. My apologies.

**Comment by mika-f** [comment]
Thanks!!

**Comment by mika-f** [comment]
Released in v3.4.0!
https://github.com/mika-f/blender-drag-and-drop/releases/tag/v3.4.0

**Pull request merged**